src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map((todo: Todo) => (
+      <TodoInfo todo={todo} />
```

Comment:
don't forget to use `key` once you're rendering the list of elements
```suggestion
      <TodoInfo key={todo.id} todo={todo} />
```